.github/ISSUE_TEMPLATE/content_suggestion.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Content Suggestion
+about: Suggest some content to live in the Devhub
+
+---
+Please fill in this form with your suggested content:
+> eg:
+>
+> Where does the content live (Github or a Website)?
+> github
+>
+> What is the url to the content (if github, the url to the file on github is needed)?
+> https://github.com/bcgov/devhub-app-web/blob/master/README.md
+>
+> What topic or journeys should this content belong too (comma seperated)?
+>
+> Community and Events, Developer Toy box, Agile Delivery Journey 
+
+__Where does the content live (Github or a Website)?__
+
+*replace with answer*
+
+__What is the url to the content (if github, the url to the file on github)?__
+
+*replace with answer*
+
+__What Topic or Journeys should this content belong too (comma separated)?__
```

Comment:
typo: too -> to

Reply:
where can one find the available topics/journeys?

Reply:
graciass